docs/blueprints/EP024.rst

```diff
@@ -0,0 +1,432 @@
+:EP: 24
+:Title: Managed resources consistency checks
+:Authors:
+    - Antonio Francisco <antonio@ansp.br>
+    - Italo Valcy <idasilva@fiu.edu>
+    - Jeronimo Bezerra <jbezerra@fiu.edu>
+    - Vinicius Arcanjo <vindasil@fiu.edu>
+:Created: 2021-11-01
+:Kytos-Version:
+:Status: Draft
+
+********************************************
+EP024 - Managed resources consistency checks
+********************************************
+
+
+Abstract
+========
+
+This document will describe some existing shortcomings of the current consistency checks on `flow_manager <https://github.com/kytos-ng/flow_manager>`_, and propose some improvements and also general development and usage guidelines.
```

Comment:
```suggestion
This document will describe some existing shortcomings of the current consistency checks on `flow_manager <https://github.com/kytos-ng/flow_manager>`_, and propose some improvements and also general development and usage guidelines. This document extends the `previous Kytos consistency documentation  <https://docs.kytos.io/developer/consistency/>`.
```

Reply:
Excellent point. It'll be handy for future readers. Just committed the suggested patch.